cmd/vic-machine/configure/configure.go

```diff
@@ -97,7 +99,7 @@ func (c *Configure) Flags() []cli.Flag {
 
 	// flag arrays are declared, now combined
 	var flags []cli.Flag
-	for _, f := range [][]cli.Flag{target, id, compute, iso, cNetwork, proxies, util, debug} {
+	for _, f := range [][]cli.Flag{target, id, compute, iso, cNetwork, proxies, dns, util, debug} {
```

Comment:
as @anchal-agrawal mentioned for create command, please also use same option order with create here